source/class/qx/Bootstrap.js

```diff
@@ -881,6 +881,20 @@ qx.Bootstrap.define("qx.Bootstrap", {
       );
     },
 
+    /**
```

Comment:
`isPojo` does not need to be in `qx.Bootstrap`, which is effectively an internal class that exists in order to work around dependency management issues - IMHO it should move to `qx.lang.Type`